include/fmt/color.h

```diff
@@ -246,7 +246,7 @@ struct color_type {
   }
   FMT_CONSTEXPR color_type(rgb rgb_color) FMT_NOEXCEPT
     : is_rgb(true), value{} {
-    value.rgb_color = (rgb_color.r << 16) + (rgb_color.g << 8) + rgb_color.b;
+    value.rgb_color = (static_cast<uint32_t>(rgb_color.r) << 16) | (static_cast<uint32_t>(rgb_color.g) << 8) | rgb_color.b;
```

Comment:
Please break the long line (https://google.github.io/styleguide/cppguide.html#Line_Length).

Reply:
ok